src/Docker/Runner.php

```diff
@@ -344,18 +345,19 @@ public function run(array $jobDefinitions, $action, $mode, $jobId, UsageFileInte
 
     private function waitForStorageJobs(array $outputs)
     {
-        $jobIds = [];
+        $tableQueues = [];
         foreach ($outputs as $output) {
             /** @var Output $output */
-            if ($output->getStorageJobIds()) {
-                $jobIds = array_merge($jobIds, $output->getStorageJobIds());
+            if ($output->getTableQueue()) {
+                $tableQueues[] = $output->getTableQueue();
             }
         }
-        $this->loggersService->getLog()->info(sprintf('Waiting for %s storage jobs.', count($jobIds)));
-        foreach ($jobIds as $jobId) {
-            $job = $this->storageClient->waitForJob($jobId);
-            if ($job['status'] == 'error') {
-                throw new UserException('Failed to process output mapping: ' . $job['error']['message']);
+        $this->loggersService->getLog()->info(sprintf('Waiting for %s Storage batches.', count($tableQueues)));
```

Comment:
batches? novej termín? `jobs` mi přijde celkem jasný.

Reply:
protoze to bohuzel neni jobs ale pocet front, v podstate je to teda vlastne pocet config rows, pocet jobu by se musel secist ze vsech front dohromady

Reply:
mozna bych tam moh napsat "Waiting for universe to calm down" ?

Reply:
až naprší a uschne! :-)

Reply:
Byl bych pro to sečtení, protože uživatel vůbec neví, co je batch. V UI mu ty processing joby můžem ukázat a to číslo batchů a jobů nebude vůbec odpovídat.

Reply:
no to ted nesectu, protoze to neni v OM public, takze zatim bez cisla a pak https://github.com/keboola/docker-bundle/pull/389#discussion_r249749295

Reply:
Ok!